lib/fizzy/execute.cpp

```diff
@@ -898,6 +898,12 @@ ExecutionResult execute(
         }
         case Instr::memory_grow:
         {
+            if constexpr (MeteringEnabled)
+            {
+                if ((ctx.ticks -= get_grow_memory_cost(stack.top().as<uint32_t>())) < 0)
```

Comment:
I would maybe define `const auto delta = stack.top().as<uint32_t>();` to use here and below

Reply:
Using `delta_pages` as that is used in both `grow_memory` and `get_grow_memory_cost`.